docs/Point SDK/Xamarin/iOS.md

```diff
@@ -140,19 +136,6 @@ The following is a checklist of the source code implementations that will assist
 BDLocationManager.Instance().GeoTriggeringEventDelegate = myDelegateImplementation;
 BDLocationManager.Instance().TempoTrackingDelegate = myDelegateImplementation;
 ```
-
-**3\. iOS Caveats**
-
-To avoid unexpected behavior in the app, implementations of the following two selectors must be present in the [Application Delegate](https://developer.apple.com/library/ios/documentation/uikit/reference/uiapplicationdelegate_protocol/Reference/Reference.html):
-
-```csharp
-public override void DidEnterBackground(UIApplication application){ ... }
```

Comment:
To confirm, these are no longer necessary?

Reply:
This is the requrements I got from our doc: https://www.notion.so/Dev-Docs-update-for-SDK-16-32fbc3b2996c4ac1bcb767b3d3ccf801
Could you please conform if we need to investigate it or just remove?
<img width="739" alt="Screenshot 2024-04-17 at 3 31 32 pm" src="https://github.com/Bluedot-Innovation/documentation/assets/24617762/788c81d2-1f08-444d-b502-0aa7fe7d1ee3">